### PR TITLE
add r-actuar

### PR DESCRIPTION
**Pull request by FernandezMathieu**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [ ] Source is from official source
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by FernandezMathieu** [comment]
Requirement R-expint in an other [pull request](https://github.com/conda-forge/staged-recipes/pull/13117)

File: recipes/r-actuar/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    - r-expint
+
+test:
+  commands:
+    - $R -e "library('actuar')"           # [not win]
+    - "\"%R%\" -e \"library('actuar')\""  # [win]
+
+about:
+  home: https://gitlab.com/vigou3/actuar
+  license: GPL-2
```
Comment (dbast) [inline recipes/r-actuar/meta.yaml:46]:
```suggestion
  license: GPL-2.0-or-later
```

File: recipes/r-actuar/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    distributions; the Poisson-inverse Gaussian discrete distribution; zero-truncated
+    and zero-modified extensions of the standard discrete distributions. Support for
+    phase-type distributions commonly used to compute ruin probabilities.'
+  license_family: GPL2
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-2'
+
+extra:
+  recipe-maintainers:
+    - FernandezMathieu
```
Comment (dbast) [inline recipes/r-actuar/meta.yaml:60]:
```suggestion
    - conda-forge/r
    - FernandezMathieu
```

**Comment by FernandezMathieu** [comment]
@conda-forge-admin, please restart ci

**Comment by FernandezMathieu** [comment]
Sound like Windows env have not been updated with r-expint for Windows jobs.
The weird part is the OSX/Linux builds throwing an r-base version error while no requirements for r-base are writen in meta.yaml.

@dbast

**Comment by dbast** [comment]
@conda-forge-admin, please restart ci

**Comment by dbast** [comment]
@FernandezMathieu Not all r-expint uploads were available on last ci retrigger... Should be the case now.

**Pull request merged**